src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,87 @@
+# Integrate your chain
+
+## Prerequisites
+
+- Make sure you have `wasm` enabled.
+- You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). Run `--gas auto --gas-adjustment 2` to provide your transactions with sufficient gas.
+
+## Devnet information
+
+- Devnet RPC: `http://devnet.rpc.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet.rpc.axelar.dev:1317`
+- Devnet gRPC: `http://devnet.rpc.axelar.dev:9090`
+- Chain ID: `devnet-wasm`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/e6f549de873e0ad211a073218f172d03f6e92c3b/axelar-chains-config/info/devnet-amplifier.json).
+
+## Set up your CosmWasm devnet environment
+
+1. Download the pre-built [`axelard v0.35.5` binary](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) from GitHub.
+1. Generate an account by running `./bin/axelard keys add test`.
+1. Fund your devnet account with `uwasm` tokens:
+    - Join the [Axelar Discord](https://discord.com/invite/axelar).
+    - Get the Developer Role.
+    - Go to the **#faucet** channel and submit a request with your account to get 100 test tokens. A sample request: `!faucet devnet axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th`
```

Comment:
Will this actually fund on the correct devnet?

Reply:
needs to be `devnet-amplifier`

Reply:
Does this mean I should change `faucet devnet axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th` to `!faucet devnet amplifier`?

Reply:
see the example Steven linked to in Discord for usage

Reply:
I think that the sample request is correct (it's not supposed to be a literal copy and paste, but an example to go off of).

Reply:
I don't understand. Why provide a sample, when we can provide the exact chain name. Just `devnet` refers to the previous wasm devnet, not the new one. We'll just get questions.

We should update the request to `!faucet devnet-amplifier [your axelar wallet address]`

Reply:
Good point. Updated.